README.md

```diff
@@ -82,6 +78,22 @@ print cursor.fetchone()
 
 For further information on getting started with NuoDB, please refer to the [NuoDB wiki](http://doc.nuodb.com/display/DOC/Getting+Started) 
 
+### Testing
```

Comment:
Can you include information about the the setup here? e.g. 

Before running the tests, ensure that: 
1. You have an unpeered broker running on localhost with a domain password of bird 
2. All database processes are shutdown.